modules/gcp_nodepool/gcp-nodepool.json

```diff
@@ -43,6 +43,16 @@
       "description": "A boolean specifying whether to use [preemptible instances](https://cloud.google.com/compute/docs/instances/preemptible)\nfor the default nodegroup or not. The preemptible instances will be configured to have the max price equal to the on-demand\nprice (so no danger of overcharging). _WARNING_: By using preemptible instances you must accept the real risk of frequent abrupt\nnode terminations and possibly (although extremely rarely) even full blackouts (all nodes die). The former is a small\nrisk as containers of Opta services will be automatically restarted on surviving nodes. So just make sure to specify\na minimum of more than 1 containers -- Opta by default attempts to spread them out amongst many nodes.\n",
       "default": false
     },
+    "guest_accelerator_type": {
+      "type": "string",
+      "description": "Guest accelerator type to add to the nodes in this nodegroup.",
```

Comment:
nit: Looks like GCP calls these node pools, not node groups?

Reply:
Following opta's convention for consistency: https://github.com/unionai/opta/blob/8fc65081c3a85091c941985a32e2ac8cbf6511e7/modules/gcp_nodepool/gcp-nodepool.yaml#L71

Reply:
Ha, super confusing but sounds reasonable :)